### ISSUE TITLE
Intuitive help for procs

### ISSUE THREAD

**Issue by olttwa** [gojektech/proctor#32]
As a user,
I want procs help to be more intuitive.

Currently, If I fetch all procs using `proctor list`, and I need help for `run-sample` proc, I need to type command: `proctor describe run-sample`. That's unintuitive.

Instead, `proctor help run-sample` or `proctor run-sample --help` would be more intuitive for displaying help.

**Comment by olttwa** [comment]
@raeoks @vjdhama thoughts? ^^

**Comment by detj** [comment]
@olttwa Yes, that'd better. Also support `proctor --help`. Right now, running `proctor --help` throws the error `Error reading proctor config`. Help routine should not check for presence of config.

Do we have a `proctor --version` command ?

**Comment by vjdhama** [comment]
@olttwa I prefer `proctor run-sample --help` over `proctor help run-sample` just because that's kind of unspoken standard for checking options of a subcommand and people expect it to be present by default.
